Kickstarter-iOS/Views/Cells/SettingsPrivacySwitchCell.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import Library
+import Prelude
+
+protocol SettingsPrivacySwitchCellDelegate: class {
+  func privacySettingsSwitchCell(_ cell: SettingsPrivacySwitchCell,
+                                 didTogglePrivacySwitch on: Bool)
+}
+
+final class SettingsPrivacySwitchCell: UITableViewCell, ValueCell, NibLoading {
+  @IBOutlet fileprivate weak var lineLayer: UIView!
+  @IBOutlet fileprivate weak var primaryDescriptionLabel: UILabel!
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
+  @IBOutlet fileprivate weak var topLineLayer: UIView!
```

Comment:
Instead of create `lineLayer` and `topLineLayer` IBOutlets, you can ctrl + drag both views and creating a `Outlet Connection`, since they have the same style. You then will have `@IBOutlet fileprivate var separatorViews: [UIView]!` and style them using:
`_ = self.separatorViews
      ||> settingsSeparatorStyle`